pkg/kubelet/cm/memorymanager/policy_static.go

```diff
@@ -767,3 +767,24 @@ func findBestHint(hints []topologymanager.TopologyHint) *topologymanager.Topolog
 	}
 	return &bestHint
 }
+
+// GetAllocatableMemory returns the amount of allocatable memory for each NUMA node
+func (p *staticPolicy) GetAllocatableMemory(s state.State) []state.Block {
+	var allocatableMemory []state.Block
+	machineState := s.GetMachineState()
```

Comment:
this is safe to be called concurrently, right?

Reply:
Yes, it has a lock inside of it
https://github.com/kubernetes/kubernetes/blob/0abb908638dcea611253ed1dfc6d66e8b31d42d4/pkg/kubelet/cm/memorymanager/state/state_checkpoint.go#L103